formlibrary/views.py

```diff
@@ -492,17 +492,6 @@ def get_queryset(self):
             training__pk=pk
         ).prefetch_related('training')
 
-    # def get(self, request, *args, **kwargs):
```

Comment:
Also remove the line 490.
i.e.
```
    def get_queryset(self):
        pk = int(self.kwargs['pk'])
``` to 
```
def get_queryset(self, pk):
```

Reply:
`get_queryset` takes no parameters. I can't override since it's a method of `ListAPIView`.